examples/server/server.cpp

```diff
@@ -1940,6 +1947,12 @@ struct server_context {
                             send_final_response(slot);
                             continue;
                         }
+                        // context shift is disabled and prompt is too large - discard it
+                        if (!params.ctx_shift && slot.n_prompt_tokens > slot.n_ctx ){
+                            slot.release();
+                            send_error(slot, "Input is too large to process. Either disable context shift or increase context length. ", ERROR_TYPE_SERVER);
```

Comment:
Shouldn't the error say enable context shift, since it's already disabled?

Reply:
Yes, I made the change. Thanks for the correction.